presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveTypeTranslator.java

```diff
@@ -227,6 +229,12 @@ public static TypeSignature toTypeSignature(TypeInfo typeInfo)
 
     @Nullable
     public static Type fromPrimitiveType(PrimitiveTypeInfo typeInfo)
+    {
+        return fromPrimitiveType(typeInfo, DEFAULT_PRECISION);
```

Comment:
`public static Type fromPrimitiveType(PrimitiveTypeInfo typeInfo)` should be deprecated now, right?